.github/workflows/test.yml

```diff
@@ -20,10 +20,10 @@ jobs:
           architecture: x64
 
       - name: Install Tox
-        run: pip install tox
+        run: pip install tox tox-py
 
       - name: Run tests
-        run: tox -e ${{matrix.python-version}} -- -vrsx --color=yes
+        run: tox --py current -- -vrsx --color=yes
```

Comment:
I don't know what is this for, but it has the net result of running no test at all. Could you please revert it? :) https://github.com/dvarrazzo/py-setproctitle/pull/102/checks?check_run_id=3769535628